packages/storage/amplify_storage_s3_dart/lib/src/storage_s3_service/service/storage_s3_service_impl.dart

```diff
@@ -297,7 +257,7 @@ class StorageS3Service {
     final urlRequest = AWSHttpRequest.raw(
       method: AWSHttpMethod.get,
       host: host,
-      path: keyToGetUrl,
+      path: '/$resolvedPath',
```

Comment:
is this supposed to get a "/" at the beginning?

Reply:
Yes. In this case we are building a url in the form of `htps://<bucket>/<path>`. It would not be a valid URL without a `/`. This does not become part of the key.

This `/` was already included by the way, I just moved the location. It was added on line 281 or 289 in the removed diff.